homeassistant/util/async_.py

```diff
@@ -65,8 +65,10 @@ def _chain_future(source, destination):
     if not isinstance(destination, (Future, concurrent.futures.Future)):
         raise TypeError('A future is required for destination argument')
     # pylint: disable=protected-access
-    source_loop = source._loop if isinstance(source, Future) else None
-    dest_loop = destination._loop if isinstance(destination, Future) else None
+    source_loop = source._loop if isinstance(  # type: ignore
```

Comment:
Inline if statements that span multiple lines should be rewritten as normal if…else.

Reply:
Done